src/components/Question.jsx

```diff
@@ -315,13 +318,8 @@ export default class Question extends React.Component {
   }
 
   getShowIrrelevantClassname(question) {
-    const debugMode = this.context.options.debugMode;
-    const startingQuestionId = this.context.options.startingQuestionId;
-    const subQuestion = question[Constants.HAS_SUBQUESTION];
-
     if (
-      (debugMode ||
-        JsonLdObjectUtils.checkId(subQuestion, startingQuestionId)) &&
+      this.isDebugged(question[Constants.HAS_SUBQUESTION]) &&
```

Comment:
No sure how this can work if `isDebugged` method works with question and here you put array of questions.